pkg/proxy/iptables/proxier.go

```diff
@@ -1160,8 +1160,8 @@ func (proxier *Proxier) syncProxyRules() {
 
 		// Capture load-balancer ingress.
 		fwChain := svcInfo.serviceFirewallChainName
-		for _, ingress := range svcInfo.LoadBalancerIPStrings() {
-			if ingress != "" {
+		for _, ingress := range svcInfo.LoadBalancerIngress() {
```

Comment:
For future - it might have made these reviews a touch easier if there was a single "prefactoring" commit that made this (effectively no-op) structural change, and then the "real" commit would have been that much easier to see.

Reply:
Noted! (I can still split it if needed 😄 )

Reply:
@Sh4d1 ,

One of my colleagues has a vested interest in moving this forward. Is it possible to go ahead and split things up as @thockin suggested? I imagine when he is able to conduct a Prod Readiness review that he'll need to start from scratch anyway due to the nearly two year gap in between the above comment and now. Since it would make it easier, maybe split them? I can help too if you like.